src/openforms/logging/tests/test_admin.py

```diff
@@ -85,3 +96,46 @@ def test_deleted_submission_doesnt_crash_logs(self):
         response = self.app.get(url, user=user)
 
         self.assertEqual(200, response.status_code)
+
+
+class TimelineLogExportsTest(TestCase):
+    def test_bare_timelinelog_export(self):
+        user = StaffUserFactory.create()
+        bare_log = TimelineLogProxyFactory.create(user=user)
+
+        dataset = TimelineLogProxyResource().export()
+
+        self.assertEqual(
+            json.loads(dataset.json),
+            [
+                {
+                    "message": bare_log.message().strip(),
```

Comment:
it's weird to assert the exact same code as the implementation. I much prefer setting up the actual test data in the factory and asserting the produced content. Tests need to cover the outcome, not the implementation.

Reply:
why is the `bare_log.message().strip()` still here instead of a literal string?